ctk/common/src/main/kotlin/org/codice/compliance/verification/profile/SingleSignOnProfileVerifier.kt

```diff
@@ -27,39 +30,45 @@ import org.codice.compliance.SAMLProfiles_4_1_4_2_i
 import org.codice.compliance.SAMLProfiles_4_1_4_2_j
 import org.codice.compliance.SAMLProfiles_4_1_4_2_k
 import org.codice.compliance.attributeNode
+import org.codice.compliance.attributeNodeNS
 import org.codice.compliance.attributeText
 import org.codice.compliance.children
+import org.codice.compliance.recursiveChildren
 import org.codice.compliance.utils.ASSERTION
+import org.codice.compliance.utils.ASSERTION_NAMESPACE
 import org.codice.compliance.utils.AUDIENCE
 import org.codice.compliance.utils.AUTHN_STATEMENT
 import org.codice.compliance.utils.BEARER
 import org.codice.compliance.utils.ENTITY
 import org.codice.compliance.utils.FORMAT
+import org.codice.compliance.utils.HOLDER_OF_KEY_URI
+import org.codice.compliance.utils.KEY_INFO_CONFIRMATION_DATA_TYPE
 import org.codice.compliance.utils.SUBJECT
 import org.codice.compliance.utils.SUBJECT_CONFIRMATION
+import org.codice.compliance.utils.SUBJECT_CONFIRMATION_DATA
 import org.codice.compliance.utils.TestCommon.Companion.REQUEST_ID
 import org.codice.compliance.utils.TestCommon.Companion.currentSPIssuer
 import org.codice.compliance.utils.TestCommon.Companion.getServiceUrl
 import org.codice.compliance.utils.TestCommon.Companion.idpMetadata
+import org.codice.compliance.utils.XSI
 import org.codice.compliance.utils.determineBinding
 import org.codice.compliance.verification.core.SubjectComparisonVerifier
 import org.codice.security.saml.SamlProtocol.Binding.HTTP_POST
 import org.codice.security.saml.SamlProtocol.Binding.HTTP_REDIRECT
-import org.opensaml.saml.saml2.core.AuthnRequest
 import org.opensaml.saml.saml2.metadata.impl.EntityDescriptorImpl
 import org.w3c.dom.Node
 
-class SingleSignOnProfileVerifier(private val authnRequest: AuthnRequest,
-                                  private val samlResponseDom: Node) {
+@Suppress("TooManyFunctions")
+class SingleSignOnProfileVerifier(private val samlResponseDom: Node) {
     /**
      * Verify response against the Core Spec document
      * 4.1.4.2 <Response> Usage
      */
     fun verify() {
-        verifyIssuer()
+        verifyIssuer(samlResponseDom)
```

Comment:
Passing in `samlResponseDom` because on line 132 it's used to validate assertion's issuer